pkg/networkmanager/dialogs-common.jsx

```diff
@@ -198,10 +201,27 @@ export const NetworkAction = ({ buttonText, iface, connectionSettings, type }) =
         if (type == 'vlan') settings = getVlanGhostSettings();
         if (type == 'team') settings = getTeamGhostSettings({ newIfaceName });
         if (type == 'bridge') settings = getBridgeGhostSettings({ newIfaceName });
+        if (type == 'wireguard') settings = getWireGuardGhostSettings({ newIfaceName });
     }
 
     const properties = { connection: con, dev, settings };
 
+    async function resolveDepsAndShowDialog(dlg) {
```

Comment:
This is a bit ugly: It does two different things, has a generic name, but is wireguard specific. My gut feeling: Name it `resolveDeps()`, pass it the `type`, make the `command -v wg` specific to `type==wireguard`, and drop the Dialogs.show(). Return a promise to indicate success/failure.